tests/test_common_usage.py

```diff
@@ -6,7 +6,7 @@
 
 def test_common_RISE_image_pipeline():  # noqa: N802 ignore case
     """No errors thrown while creating a relevance map and visualizing it."""
-    input_image = np.random.random((5, 5, 3))
+    input_image = np.random.random((224, 224, 3))
```

Comment:
This would be a regression from https://github.com/dianna-ai/dianna/pull/606 which we merged yesterday :-)

```suggestion
    input_image = np.random.random((5, 5, 3))
```